forgettable-frontend/src/index.tsx

```diff
@@ -16,7 +16,6 @@ import Encounters from './pages/encounters/encounters';
 ReactDOM.render(
     <React.StrictMode>
       <BrowserRouter>
-        <NavBar />
```

Comment:
Ah, also. Please remove this conditionally, otherwise it's going to disappear from all other pages. Let me know if you don't know how :)

Reply:
How would I be able to do that?

Reply:
> How would I be able to do that?

Would it be something like [this](https://tanzu.vmware.com/content/blog/showing-and-hiding-conditional-html-without-javascript)?